src/Fantomas.Tests/KeepIndentInBranch.fs

```diff
@@ -0,0 +1,253 @@
+﻿module Fantomas.Tests.KeepIndentInBranch
+
+open NUnit.Framework
+open FsUnit
+open Fantomas.Tests.TestHelper
+open Fantomas.FormatConfig
+
+let config =
+    { config with
+          KeepIndentInBranch = true }
+
+[<Test>]
+let ``single expression in if branch, multiple expressions in else branch`` () =
+    formatSourceString
+        false
+        """
+if someCondition then
+    0
+else
+    let config = Configuration.Read "/myfolder/myfile.xml"
+    let result = Process.main config otherArg
+    if result.IsOk then
+        0
+    else
+        -1
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+if someCondition then
+    0
+else
+let config =
+    Configuration.Read "/myfolder/myfile.xml"
+
+let result = Process.main config otherArg
+if result.IsOk then 0 else -1
```

Comment:
I think some of the value of this test case has been lost with how short `0` and `-1` are. Maybe it's worth making these expressions long enough that Fantomas splits them onto new lines, so we can see how they behave?